matlab/MeCorr/MATLAB_CODE_methyl.m

```diff
@@ -87,16 +86,14 @@
     for i = 2:length(unqgenes)
         
         modeltemp = acetylation_model ;
-         modeltemp = deleteModelGenes(modeltemp,unqgenes(i));
+        modeltemp = deleteModelGenes(modeltemp,unqgenes(i));
         model3 = modeltemp;
-      
         
         [solf.x,sol11] =  constrain_flux_regulation(model3,[],[],0,0,0,[],[],minfluxflag);
         if ~isempty(solf.x) && ~isnan(solf.x)
             acet_screen_rpmi(i,2) = solf.x(objpos); % impact on growth

```

Comment:
Change variable name so it's more obvious that it's the growth rate from the constrained metabolic model. Also, are you intentionally using the acetylation model? It is confusing since you're storing the objective coefficient value into the variable name `epsilon_methylation`.